### ISSUE TITLE
Link styles on generic pages are not accessible

### ISSUE THREAD

**Issue by mandymichael** [mandymichael/variable-fonts-dev#3]
The colour contrast on link styles in the generic pages are not accessible. These should be updated to a more appropriate colour.

**Comment by mandymichael** [comment]
I made this way more generic across the site and just switched it to black and white, it's a lot better now.
